Formula/chronograf.rb

```diff
@@ -22,22 +22,16 @@ class Chronograf < Formula
   depends_on "kapacitor"
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV.prepend_create_path "PATH", buildpath/"bin"
     Language::Node.setup_npm_environment
-    chronograf_path = buildpath/"src/github.com/influxdata/chronograf"
-    chronograf_path.install buildpath.children
 
-    cd chronograf_path do
-      cd "ui" do # fix compatibility with the latest node
-        system "yarn", "upgrade", "parcel@1.11.0"
-      end
-      system "make", "dep"
-      system "make", ".jssrc"
-      system "make", "chronograf"
-      bin.install "chronograf"
-      prefix.install_metafiles
+    cd "ui" do # fix compatibility with the latest node
+      system "yarn", "upgrade", "parcel@1.11.0"
     end
+    system "make", "dep"
+    system "make", ".jssrc"
+    system "make", "chronograf"
+    bin.install "chronograf"
+    prefix.install_metafiles
```

Comment:
I believe this line is not necessary.

https://github.com/Homebrew/brew/blob/a95dd7d48588099fae744cf8970c9e7ea25368bd/Library/Homebrew/build.rb#L159